spyvm/primitives.py

```diff
@@ -1588,9 +1588,27 @@ def func(interp, s_frame, w_block_closure, w_a0):
 CTXT_AT_PUT = 211
 CTXT_SIZE = 212
 
+@expose_primitive(CTXT_SIZE, unwrap_spec=[object])
+def func(interp, s_frame, w_rcvr):
+    from spyvm.interpreter import Return
+    from spyvm.error import MethodNotFound
+    try:
+        s_frame._sendSelector(interp.image.w_numTemps, 0, interp, w_rcvr,
+                              w_rcvr.class_shadow(interp.space))
+    except Return, ret:
+        # numTemps should always return
+        ret_value = ret.value
+        if isinstance(ret_value, model.W_SmallInteger):
+            return interp.space.wrap_int(ret_value.value + 1)
```

Comment:
CTXT_SIZE should return number of temps + 1 (for home context) as far as I know.

E.g.: `thisContext basicSize` is `1` but `thisContext numTemps` is `0`.